cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.9.0
+
+_Released 03/28/2023 (PENDING)_
+
+**Misc:**
```

Comment:
This feels a bit more like a bug fix to me

Reply:
or an enhacement to the debug experience. Not sure misc is the best section 🤷🏻‍♀️

Reply:
yeah that makes sense to me. I updated it to be under bugfixes